lib/round.rb

```diff
@@ -0,0 +1,76 @@
+class Round
+  # require 'turn'
+  attr_reader :deck, :turns
+
+  def initialize(deck)
+    @deck = deck
+    @turns = []
+    @guess = 0
+  end
+
+  def current_card
+    return deck.cards[@guess]
+  end
+
+  def take_turn(guess)
+      turn = Turn.new(guess, current_card)
+      @turns << turn
+      @guess += 1
```

Comment:
Since you're incementing guess each time you shovel in a turn object, you may consider removing the need for the guess instance variable since you could just call `@turns.count` whenever you need to know how many guesses there are.